static/app/views/issueList/taxonomies.tsx

```diff
@@ -19,19 +22,29 @@ export const ISSUE_TAXONOMY_CONFIG: Record<
     categories: [IssueCategory.ERROR, IssueCategory.OUTAGE],
     label: t('Errors & Outages'),
     key: 'errors-outages',
+    description: t(
+      'Issues that break functionality, such as application errors, failed jobs, or downtime incidents.'
```

Comment:
```suggestion
      'Issues that break functionality such as application errors, failed jobs, or downtime incidents.'
```

super nit, just noticed that this one added a comma after "such" and breached metrics didn't